tokio-epoll-uring/src/system/lifecycle.rs

```diff
@@ -36,18 +36,46 @@ unsafe impl Sync for System {}
 
 static SYSTEM_ID: std::sync::atomic::AtomicUsize = std::sync::atomic::AtomicUsize::new(0);
 
+#[derive(Debug)]
+pub enum LaunchResult {
+    IoUringBuild(std::io::Error),
+}
+
+impl std::error::Error for LaunchResult {
+    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+        match self {
+            LaunchResult::IoUringBuild(e) => Some(e),
+        }
+    }
+}
+
+impl std::fmt::Display for LaunchResult {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let (prefix, e) = match &self {
+            LaunchResult::IoUringBuild(e) => ("build io_uring instance", e),
+        };
+        if f.alternate() {
+            write!(f, "{prefix}: {e:#}")
```

Comment:
I don't think the alternate has any meaning for std (funnily enough) ... It would be great to have a way to "forward it".